<ui version="4.0" >
 <author>Peter Hedlund</author>
 <class>GeneralOptionsBase</class>
 <widget class="QWidget" name="GeneralOptionsBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>383</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="KButtonGroup" name="kbuttongroup" >
     <property name="title" >
      <string>Open/Save</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="kcfg_AutoOpenLast" >
        <property name="toolTip" >
         <string>Always open the collection that was active when quitting Parley</string>
        </property>
        <property name="whatsThis" >
         <string>When this option is checked, the last active collection will be loaded automatically when starting Parley instead of showing the welcome screen.</string>
        </property>
        <property name="text" >
         <string>Always load the last opened collection on start</string>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_AutoSave" >
        <property name="toolTip" >
         <string>Allow automatic saving of your work</string>
        </property>
        <property name="whatsThis" >
         <string>Your work will be automatically saved if you check this option</string>
        </property>
        <property name="text" >
         <string>&amp;Save vocabularies automatically on close and quit</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QCheckBox" name="kcfg_AutoBackup" >
          <property name="toolTip" >
           <string>Allow automatic saving of your work</string>
          </property>
          <property name="whatsThis" >
           <string>Your work will be automatically saved if you check this option</string>
          </property>
          <property name="text" >
           <string>&amp;Create a backup every</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KIntSpinBox" name="kcfg_BackupTime" />
        </item>
        <item>
         <widget class="QLabel" name="lblBackupInterva" >
          <property name="text" >
           <string>minutes</string>
          </property>
          <property name="buddy" >
           <cstring>kIntSpinBox1</cstring>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>91</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_sep" >
          <property name="text" >
           <string>Sep&amp;arator for text file import/export (CSV):</string>
          </property>
          <property name="buddy" >
           <cstring>kcfg_SeparatorCombo</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="kcfg_SeparatorCombo" >
          <property name="minimumSize" >
           <size>
            <width>140</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Choose which separator you want to use to separate your data when importing and exporting text.</string>
          </property>
          <property name="whatsThis" >
           <string>Choose the separator to use to divide the parts of an expression when importing or exporting data as text.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>121</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="KButtonGroup" name="editingGroup" >
     <property name="title" >
      <string>Editing</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="kcfg_AutoEntryApply" >
        <property name="toolTip" >
         <string>Your changes will be applied automatically.</string>
        </property>
        <property name="whatsThis" >
         <string>If checked, you will not be asked if you really want this change; it will be applied automatically.</string>
        </property>
        <property name="text" >
         <string>Appl&amp;y changes without asking</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="kcfg_SmartAppend" >
        <property name="toolTip" >
         <string>If checked, the entry dialog will popup repeatedly</string>
        </property>
        <property name="whatsThis" >
         <string>If this feature is activated, you will repeatedly be prompted with the entry dialog. After entering the first original you have to enter the corresponding translations. Afterwards you proceed with the next original and its translations until you stop by pressing the ESC key.</string>
        </property>
        <property name="text" >
         <string>Smart a&amp;ppending</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>558</width>
       <height>81</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_AutoSave</tabstop>
  <tabstop>kcfg_AutoBackup</tabstop>
  <tabstop>kcfg_BackupTime</tabstop>
  <tabstop>kcfg_SmartAppend</tabstop>
  <tabstop>kcfg_AutoEntryApply</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
